### PR TITLE
[CDAP-1006] Replaced Id.Account with Id.Namespace

### PR DESCRIPTION
**Pull request by bdmogal**
Part 1 of replacing ``accountId`` with ``namespaceId``.

Looks long, but essentially only replaces ``Id.Account`` with ``Id.Namespace`` in this PR.

Will handle other ``accountId`` -> ``namespaceId`` replacements in separate PRs.

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java [hunk 1]
```diff
@@ -153,7 +152,8 @@ public AppFabricHttpHandler(Authenticator authenticator, CConfiguration configur
     this.queueAdmin = queueAdmin;
     this.txClient = txClient;
     this.dsFramework =
-      new NamespacedDatasetFramework(dsFramework, new DefaultDatasetNamespace(configuration, Namespace.USER));
+      new NamespacedDatasetFramework(dsFramework,
+                                     new DefaultDatasetNamespace(configuration, co.cask.cdap.data.Namespace.USER));
```
Comment (anwar6953) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java]: Why fully qualified? (and removed import)?

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java [hunk 2]
```diff
@@ -91,14 +90,14 @@ public LocalManager(CConfiguration configuration, PipelineFactory pipelineFactor
     this.programTerminator = programTerminator;
     this.datasetFramework =
       new NamespacedDatasetFramework(datasetFramework,
-                                     new DefaultDatasetNamespace(configuration, Namespace.USER));
+                                     new DefaultDatasetNamespace(configuration, co.cask.cdap.data.Namespace.USER));
```
Comment (anwar6953) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java]: same here. Whats wrong with importing data.Namespace? It doesn't conflict with Id.Namespace

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java (hunk 1 above)
Comment (bdmogal) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java]: Automatic thing from the IDE. will fix.

**Comment by anwar6953** [comment]
Also update Program.java getAccountId to getNamespaceId

**Comment by anwar6953** [comment]
Also wanna change variable names to avoid confusion in the future?
(Such as: AbstractAppFabricHttpHandler.java line 277)

**Comment by bdmogal** [comment]
Yes, all those changes coming up in separate PRs. This will impact a lot of other pull requests so want to do it one at a time.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java (hunk 2 above)
Comment (bdmogal) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/LocalManager.java]: again from the IDE. will fix.

**Comment by bdmogal** [comment]
@anwar6953 addressed comments. Please review.

**Comment by bdmogal** [comment]
Build running here - https://builds.cask.co/browse/CDAP-DUT465-2

**Comment by anwar6953** [comment]
Changes LGTM. (With the understanding that more changes are required in Program.java, and the usages of it).

**Comment by bdmogal** [comment]
Build succeeded. Merging.

**Pull request merged**